src/Mod/AddonManager/AddonManager.py

```diff
@@ -977,6 +1044,62 @@ def dependency_dialog_yes_clicked(self, repo: AddonManagerRepo) -> None:
         self.dependency_installation_dialog.show()
         self.dependency_installation_worker.start()
 
+    def no_python_exe(self, repo: AddonManagerRepo) -> None:
+        if hasattr(self, "dependency_installation_dialog"):
+            self.dependency_installation_dialog.hide()
+        result = QtWidgets.QMessageBox.critical(
+            self.dialog,
+            translate("AddonsInstaller", "Cannot execute Python"),
+            translate(
+                "AddonsInstaller",
+                "Failed to automatically locate your Python executable, or the path is set incorrectly. "
+                "Please check the Addon Manager preferences setting for the path to Python.",
+            )
```

Comment:
IIRC these multi-line strings inside translate() were not picked up correctly by pylupdate. We need to check that again

Reply:
Ah, I meant to check on that and never got around to it. I guess from a translation perspective I have to just have it all as one long string, if I break it up it won't make any sense. I'll fix those, thanks.